circuit_knitting_toolbox/circuit_cutting/wire_cutting/mip_model.py

```diff
@@ -493,7 +493,13 @@ def solve(self, min_postprocessing_cost: float) -> bool:
             min_postprocessing_cost,
             str(type(min_postprocessing_cost)),
         )
-        self.model.export_as_lp(path="./docplex_cutter.lp")
+        try:
+            self.model.export_as_lp(path="./docplex_cutter.lp")
+        except RuntimeError:
+            print(
```

Comment:
So this is a case where the model checkpoint export failed, but we don't want to error out?

How exactly does having CPLEX intermittently cause this model export to fail?

Reply:
> How exactly does having CPLEX intermittently cause this model export to fail?

It's deep within DOcplex, but you can see the traceback in #119.

Reply:
> > How exactly does having CPLEX intermittently cause this model export to fail?
> 
> It's deep within DOcplex, but you can see the traceback in #119.

I see, is it indeed intermittent, or is it consistent behavior? In either case, do you feel like just erroring is better than trying to continue? Are we sure that something isn't corrupted when we are trying to continue (some dictionary somewhere changed size unexpectedly)

Reply:
I believe it happens consistently, and is due to some internal DOcplex error.  In fact, you can tell from the comment two lines later ("this call updates the dict so we must iterate on something else.") that they have had similar issues with the surrounding code before.  I would suspect that they are not sufficiently testing the code paths that occur when cplex is not present.  In any case, I could imagine them making a new DOcplex release that fixes this, so I do not want to rely on this as an indicator that cplex is not installed.  Plus, the whole reason we are exporting the LP file is for user convenience.  If that export fails (which could even happen because the current directory is not writable by the user), I see no reason _not_ to try to continue.

You do bring up an interesting point, which is that when this `RuntimeError` is raised, some -- but not all -- of the items will have had `resolve()` called on them.   This might cause a later problem, but the only time we know it might happen is when cplex is not installed, and if that's the case, the user is about to get an informative error saying as much, so I think it makes sense to continue to that point.